.github/workflows/codeql.yml

```diff
@@ -70,12 +57,29 @@ jobs:
           query-filters:
           - exclude:
              tags: cpp/integer-multiplication-cast-to-long
-           
+    - name: env
+      env:
+        GITHUB_CONTEXT: ${{ toJson(github) }}
+      run: |
+        env
+    - name: module list
+      shell: bash -l {0}
+      run: |
+        module list
+        printenv PATH
     - if: matrix.build-mode == 'manual'
-      name: Configure Trilinos            
+      name: Get dependencies
+      run: |
+          bash -lc "${GITHUB_WORKSPACE}/packages/framework/get_dependencies.sh --container"
+    - if: matrix.build-mode == 'manual'
+      name: Configure Trilinos
+      shell: bash -lc {0}
       run: |
           mkdir -p trilinos_build
-          cd trilinos_build 
+          cd trilinos_build
+
+          bash ${GITHUB_WORKSPACE}/packages/framework/GenConfig/LoadEnv/load-env.sh --force rhel8_gcc-openmpi_debug_shared_no-kokkos-arch_no-asan_complex_no-fpic_mpi_no-pt_no-rdc_no-uvm_deprecated-on_no-package-enables
+
           cmake -G 'Unix Makefiles' -DTrilinos_ENABLE_TESTS=OFF -DTrilinos_ENABLE_Epetra=OFF -DTrilinos_ENABLE_AztecOO=OFF -DTrilinos_ENABLE_Ifpack=OFF -DTrilinos_ENABLE_ML=OFF -D Trilinos_ENABLE_Triutils=OFF -DTrilinos_ENABLE_Tpetra=ON -DTrilinos_ENABLE_MueLu=ON -DTrilinos_ENABLE_Krino=OFF ..
```

Comment:
This needs to be changed to build the correct set of packages (the ones modified as part of this PR, but not downstream ones).

Could also use the PR scripts to facilitate this, and maybe add the configure option that turns off downstream packages (something like `Trilinos_ENABLE_DOWNSTREAM_PACKAGES` ish)?  Using GenConfig will likely fix your issues with not finding TPLs.